tests/jvm/src/test/resources/lowlevel.expect

```diff
@@ -365,6 +365,458 @@ Synthetics:
   [0:13..0:26): augmentString => _root_.scala.Predef.augmentString(Ljava/lang/String;)Ljava/lang/String;.
   [0:27..0:28): * => _star_.
 
+semanticdb/integration/src/main/scala/example/Types.scala
```

Comment:
I don't see any test for projection on TypeRef with tparams. How does something like
```
package am
trait A[T] { type Self; def self: Self }
object A {
  def <<foo>>[T] = null.asInstanceOf[A[T]].self
}
```

look like?

Reply:
Done.